src/App.tsx

```diff
@@ -9,7 +9,7 @@ export const App = () => {
   const [userName, setUserName] = useState<string>("")
   const [isUserFound, setIsUserFound] = useState<boolean>(false)
 
-  const {refetch} = rqFunctions.GetUser(userName)
+  const {refetch} = rqFunctions.useGetUser(userName)
```

Comment:
change import
`import { useGetUser } from './components/ReactQueryWrapper'`
```suggestion
  const { refetch } = useGetUser(userName)
```